Storage/RecentlyClosedTabs.swift

```diff
@@ -5,12 +5,14 @@
 import Foundation
 import Shared
 
+private let log = Logger.browserLogger
+
 open class ClosedTabsStore {
     let prefs: Prefs
 
     lazy open var tabs: [ClosedTab] = {
         guard let tabsArray: Data = self.prefs.objectForKey("recentlyClosedTabs") as Any? as? Data,
-              let unarchivedArray = NSKeyedUnarchiver.unarchiveObject(with: tabsArray) as? [ClosedTab] else {
+            let unarchivedArray = (try? NSKeyedUnarchiver.unarchiveTopLevelObjectWithData(tabsArray)) as? [ClosedTab] else {
```

Comment:
Should catch this error, rather than using `try?`. I suppose given that we'll probably still just return `[]`, this may be fine, probably leave comment, as it isn't a convention we're trying to follow 😄

Reply:
It gets a bit weird when we do it with try/catch due to the casting lol.

```swift
lazy open var tabs: [ClosedTab] = {
    guard let tabsArray: Data = self.prefs.objectForKey("recentlyClosedTabs") as Any? as? Data else { return [] }
    do {
        if let unarchivedArray = try NSKeyedUnarchiver.unarchiveTopLevelObjectWithData(tabsArray) as? [ClosedTab] {
            return unarchivedArray
        }
    } catch {
        log.error("Failed to unarchive recently closed tabs: \(error.localizedDescription)")
    }
    return []
}()
```

I'll just put a comment 😅